src/importexport/imagesexport/internal/pdfwriter.h

```diff
@@ -31,10 +31,8 @@ class PdfWriter : public notation::AbstractNotationWriter
     INJECT(iex_imagesexport, IImagesExportConfiguration, configuration)
 
 public:
-    std::vector<INotationWriter::UnitType> supportedUnitTypes() const override;
-    Ret write(const notation::INotationPtr notation, system::IODevice& destinationDevice, const Options& options = Options()) override;
-    Ret writeList(const notation::INotationPtrList& notations, system::IODevice& destinationDevice,
-                  const Options& options = Options()) override;
+    std::vector<notation::WriterUnitType> supportedUnitTypes() const override;
+    Ret write(const notation::INotationPtrList& notations, system::IODevice& destinationDevice,const Options& options = Options()) override;
```

Comment:
Uncrustify bug: if I write on one line
```
Ret write(const notation::INotationPtrList& notations, system::IODevice& destinationDevice, const Options& options = Options()) override;
```
it wants me to change it into 
```
Ret write(const notation::INotationPtrList& notations, system::IODevice& destinationDevice, 
          const Options& options = Options()) override;
```
And if I do that, it wants me to change it into 
```
Ret write(const notation::INotationPtrList& notations, system::IODevice& destinationDevice,const Options& options = Options()) override;
```
So that is what I did, although it's not beautiful without a space after that comma.

Reply:
I think, we may rename `destinationDevice` to like `destination` or `dest` or `destDevice` to correspond the line width constraint :)

Reply:
We might also change this line:
https://github.com/musescore/MuseScore/blob/c111b1d1f482dea0cc8166d072b967eb07cefa38/tools/codestyle/uncrustify_musescore.cfg#L429
I think it would be okay to have a space after all commas.

Reply:
Yes, I agree with you

Reply:
Should I go for "add" or "force"?

Reply:
I think better "force", but need testing

Reply:
Let's make a separate PR if there are many fixes

Reply:
I think "add" appears to be better. "force" would screw up things like https://github.com/musescore/MuseScore/blob/c111b1d1f482dea0cc8166d072b967eb07cefa38/src/importexport/guitarpro/internal/importgtp-gp6.cpp#L69 and https://github.com/musescore/MuseScore/blob/c111b1d1f482dea0cc8166d072b967eb07cefa38/src/notation/view/widgets/editstyle.cpp#L95
I don't think we really want that :)

Reply:
Let's use "add"

Reply:
Here's the PR: https://github.com/musescore/MuseScore/pull/7961